examples/api/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -48,6 +48,7 @@
       ignoresPersistentStateOnLaunch = "NO"
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
+      enableGPUValidationMode = "1"
```

Comment:
Can we remove this? We don't like to mess with these files.

Reply:
It has been reverted.